src/api/process.c

```diff
@@ -542,11 +731,16 @@ static int f_running(lua_State* L) {
   return 1;
 }
 
-static const struct luaL_Reg lib[] = {
+static int process_gc(lua_State *L) {
+  kill_list_wait_all(&kill_list);
+  SDL_WaitThread(kill_list_thread, NULL);
+  kill_list_free(&kill_list);
+  return 0;
+}
```

Comment:
Instead of doing this in Lua gc, we could do this in `main` and close the window in the meantime, right?

Reply:
I want to keep process stuff in Process. In my testing, it doesn't halt the closure of the window for too long.

Of course, a better fix is to decouple the window.